File: backend/src/main/java/me/performancereservation/domain/reservation/service/SeatReservationService.java
package me.performancereservation.domain.reservation.service;

import me.performancereservation.domain.reservation.dto.ReservationResponse;

public interface SeatReservationService {
    ReservationResponse reserve(Long performanceId, Long scheduleId, Long userId, int quantity);

    void cancel(Long scheduleId, Long userId);
}

File: backend/src/main/kotlin/me/performancereservation/domain/reservation/service/SeatReservationService.kt
package me.performancereservation.domain.reservation.service

import me.performancereservation.domain.reservation.dto.ReservationResponse

interface SeatReservationService {
    fun reserve(performanceId: Long, scheduleId: Long, userId: Long, quantity: Int): ReservationResponse

    fun cancel(reservationId: Long, userId: Long): Long
}
